Rocket.Chat/Views/Cells/Chat/ChatMessageImageView.swift

```diff
@@ -49,16 +50,24 @@ final class ChatMessageImageView: UIView {
             addGestureRecognizer(tapGesture)
         }
 
-        labelTitle.text = attachment.title
-
-        let imageURL = attachment.fullImageURL()
+		guard let imageURL = attachment.fullImageURL() else {
+			return
+		}
+        if imageURL.absoluteString.starts(with: "http://") {
+        	isLoadable = false
+			labelTitle.text = localized("alert.insecure_image")
```

Comment:
Maybe it's better if we keep the title like:

eg. "image.png (insecure resource not loaded)"

Reply:
But if you upload a file "Screenshot-2018-01-17-22-49.jpg (insecure resource not loaded)" you will only see "Screenshot-2018-01..." and no information, did it previously "[alert] [title]", so at least some information. I'll push that :)

Reply:
🚫 image.png

What do you think? 😅

Reply:
Uhm... As some people upload their images with weird emoji titles I don't think that this will be the better way to go 🚫🙈😄

Reply:
I think the image I created is internationally valid ;)
Otherwise feel free to suggest one. With the image added it may be okay to display it other way round ([title] [alert])